codecov.yml

```diff
@@ -21,13 +21,13 @@ component_management:
     - component_id: actual_code
```

Comment:
Why was this file changed?

Reply:
These were some black (or prettier?) updates, I can try to unstage / revert these changes but they shouldn't affect anything